solutions/security/detect-and-alert/suppress-detection-alerts.md

```diff
@@ -134,13 +130,28 @@ With alert suppression, detection alerts aren’t created for the grouped source
 
     :::{image} /solutions/images/security-timeline-button.png
     :alt: Investigate in timeline button
-    :width: 350px
+    :width: 250px
     :screenshot:
     :::
 
 * Alert details flyout — Select **Take action** → **Investigate in timeline**.
 
 
+## Impact of closing suppression alerts [security-alert-suppression-impact-close-alerts]
+
+As a best practice, avoid closing suppresssion alerts (alerts generated for alert suppression) before an active suppression window ends. Closing alerts early can interrupt alert suppression or cause unexpected changes. 
```

Comment:
-> suppressed alerts

Reply:
I would change this to something like: "By default, if you close a suppressed alert where suppression window is still active, the suppression will restart and subsequent alerts will be suppressed and added to a new alert."

Reply:
```suggestion
By default, if you close a suppressed alert while a suppression window is still active, suppression resets. Subsequently, any new qualifying alerts are suppressed and added to a new alert for suppression.
```